book/2 - Configuration Basics/2 - WebAdmin.html

```diff
@@ -0,0 +1,20 @@
+<p>
+	The WebAdmin allows you to control various aspects of Cuberite, including player permissions.
+	A typical <code>webadmin.ini</code> configuration looks like this:
+</p>
+
+<figure class="codebox">
+<pre><code>
+[User:john]
+Password=cuberiteRocks
+
+[WebAdmin]
+Ports=8080
+Enabled=1
+</code></pre>
+</figure>
+
+
+<p>
+	In the example above, you can login to the web admin using the username <code>john</code> and the password <code>cuberiteRocks</code> by pointing your browser to <code>http://localhost:8080</code>.
```

Comment:
Localhost should probably be the IP of the server. I think there's a section for finding that.